diagrams/library-gen.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
```

Comment:
Why do we commit a generated file?

Reply:
So we know when it changes. I used to be “don't commit generated files”, but I pretty much turned around on that.

Reply:
Is the explanation sufficient (for now)? I certainly don't mind culling what generated files are committed later on, though.